src/views/Hero/Hero.scss

```diff
@@ -1,11 +1,57 @@
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
+
 .HeroView {
-  height: 100%;
+  display: flex;
+  overflow: hidden;
+  justify-content: center;
+  min-height: 100%;
 
   &__homeView {
     background: linear-gradient(180deg, var(--wedgewood__gradient) 0%, var(--waterleaf) 100%);
-    height: 90%;
+    min-height: 90%;
     border-bottom-left-radius: 50px;
     border-bottom-right-radius: 50px;
     box-shadow: 0 10px 16px rgba(0, 0, 0, 0.25);
   }
+
+  &__container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
```

Comment:
It looks like each child element within this container is being sized to-scale (i.e. the widths and margins of the two child elements fill up the entire container). In this case, would we still need this property?